Project/Source/Components/ComponentTrail.h

```diff
@@ -3,49 +3,77 @@
 
 #include "Utils/Pool.h"
 #include "Utils/UID.h"
+
 #include "Math/float4.h"
 #include "Math/float2.h"
 #include "Math/float4x4.h"
 #include "Math/Quat.h"
+#include "imgui_color_gradient.h"
```

Comment:
Forward declaration for ImGradient and ImGradientMark:
.cpp
#include "imgui_color_gradient.h"

.h
delete #include "imgui_color_gradient.h"
class ImGradient;
class ImGradientMark;